pages/flipper.tsx

```diff
@@ -49,7 +49,7 @@ function Flipper(props: Props) {
             {getHeadElement(undefined, 'Free auction house item flipper for Hypixel Skyblock', undefined, ['flipper'])}
             <Container>
                 <Search />
-                <h2>Item-Flipper (WIP)</h2>
+                <h2 style={{display: "flex", alignItems: "center"}}><b>Item Flipper</b>&nbsp;<span style={{backgroundColor: "white", color: "black", padding: "4px", borderRadius: "4px",fontSize:"2vh"}}>WIP</span></h2>
```

Comment:
Remove the `b` tag. There is no reason to make this headline look different that all the others